jenkinsapi/build.py

```diff
@@ -98,7 +98,11 @@ def get_params(self):
         """
         actions = self._data.get('actions')
         if actions:
-            parameters = actions[0].get('parameters', {})
+            parameters = {}
+            for elem in actions:
+                if 'parameters' in elem:
```

Comment:
If `'parameters'` are in `elem`, then why `elem.get()` and not just `elem['parameters']`?